Gemfile

```diff
@@ -5,15 +5,15 @@ gem 'bootsnap', require: false
 gem 'listen'
 gem 'net-smtp', require: false
 gem 'psych', '< 4'
-gem 'rails', '~>7.0.8'
+gem 'rails', '~>7.2'
 gem 'sprockets', '~>3.7.2'
 #gem 'sprockets-rails', require: 'sprockets/railtie'
 gem 'sqlite3'
 # Force newer version of mail for compatibility with rails 6.0.6.1
 gem 'mail', '> 2.8.0.1'
 
 # Assets
-gem 'bootstrap', '~> 4.0'
+gem 'bootstrap', '4.6.2'
```

Comment:
Late last month they bumped Bootstrap v4 to 4.6.2.1 which removed the inclusion of the sass engine dependency `sassc-rails`. I guess that gem hasn't been maintained in several years, so bootstrap is deprecating use of it and allowing the use of other sass engine gems.

Rather than using the newest v4 Bootstrap and explicitly declaring `sassc-rails` in our Gemfile, I decided it made sense to pin to the last version that included the gem and we can check out the other sass engine gems they suggest and see what will work for us before moving away. Also, may be worth investigating whether a move to Bootstrap 5 is doable. I meant to create a ticket(s) for those but don't think I did, so thank you for the reminder.